pf/tfbridge/main.go

```diff
@@ -91,10 +92,10 @@ func handleFlags(ctx context.Context, prov ProviderInfo, meta ProviderMetadata,
 // Implements main() or a bridged Pulumi plugin, complete with argument parsing.
 //
 // This is an experimental API.
-func MainWithMuxer(ctx context.Context, pkg string, meta ProviderMetadata, infos ...Muxed) {
-	f := MakeMuxedServer(ctx, pkg, meta, infos...)
+func MainWithMuxer(ctx context.Context, schema []byte, info tfbridge.ProviderInfo) {
```

Comment:
Similarly to tfgen, consider if this can be merged into Main as a special case. If we don't like the signature of Main we can tweak that later on (just need to annotate "breaking" for the users and explain how to migrate). Proliferation of Main's that are linked to from outside packages is a bit unfortunate. Or do we need that for some reason?

Reply:
This is just about not making a breaking change. As I said replying to the next comment, my plain is to finish this PR and merge. I'll follow up with a separate PR to fix the function API.

Reply:
Perfect SGTM.